dvc/repo/experiments/show.py

```diff
@@ -177,26 +177,26 @@ def _build_rows(
                 )
 
 
-def _sort_column(
+def _sort_column(  # noqa: C901
     sort_by: str,
     metric_names: Mapping[str, Iterable[str]],
     param_names: Mapping[str, Iterable[str]],
 ) -> Tuple[str, str, str]:
-    path, _, sort_name = sort_by.rpartition(":")
+    path, _, sort_name = sort_by.partition(":")
```

Comment:
Needed to address https://github.com/iterative/dvc/pull/9819#discussion_r1287316150, but not sure if there was a reason for using `rpartition` (cc @pmrowla)

Reply:
You can have colons in filenames.